extensions/amp-bind/amp-bind.md

```diff
@@ -218,32 +278,143 @@ Only binding to the following components and attributes are allowed. Most bindab
 
 <sup>1</sup>Denotes bindable attributes that don't have a non-bindable counterpart.
 
-### Expressions
+Note that boolean expression results toggle boolean attributes.
 
-`amp-bind` expressions are similar to JavaScript with some important differences.
+```html
+<amp-video [controls]="expr"...>
+```
 
-#### Differences from JavaScript
+When `expr` evaluates to `true`, the `<amp-video>` element has the `controls` attribute. When `expr` evaluates to `false`, the `controls` attribute is removed.
```

Comment:
I think this should go in the bullet list at the bottom of the "Bindings" section.